api/src/test/java/org/pmiops/workbench/reporting/ReportingUploadServiceTest.java

```diff
@@ -127,12 +161,18 @@ public void testUploadSnapshot() {
 
     final String expandedQuery =
         QueryParameterValues.formatQuery(QueryParameterValues.replaceNamedParameters(job0));
-
     assertThat(expandedQuery).containsMatch("INSERT\\s+INTO");
+
+    assertThat(
+            (double)
+                timestampQpvToInstant(jobs.get(1).getNamedParameters().get("creation_time__0"))
+                    .toEpochMilli())
+        .isWithin(500.0)
+        .of(THEN.toEpochMilli());
   }
 
   @Test
-  public void testUploadSnapshot_batchInserts() {
+  public void testUploadSnapshot_dmlBatchInserts() {
```

Comment:
should we test the streaming version of this?

Reply:
We do, right below. There's no manual batching with the stream approach though. The BQ client does that for us I'm assuming.